lib/bolt/cli.rb

```diff
@@ -6,6 +6,7 @@
 require 'io/console'
 require 'logging'
 require 'optparse'
+require 'r10k/action/puppetfile/install'
```

Comment:
We should make sure this doesn't accidentally load Puppet. It may be worth restricting it to the install workflow where it's used.

Reply:
r10k doesn't depend on puppet so it shouldn't